repos/fdbt-admin/src/pages/Reporting.tsx

```diff
@@ -14,6 +17,10 @@ interface ReportingProps {
     isFullAdmin: boolean;
 }
 
+const mapIntoArrayOfArrays = (input: string[]): string[][] => {
```

Comment:
can we name it as mapInto2DArray or convertMapInto2DArray

Reply:
I think that would make it **less** clear what it is - I don't know what a 2D array is. The return type string[][] and the name 'array of arrays' I think is more clear.